compose/config/config.py

```diff
@@ -690,6 +749,26 @@ def resolve_volume_path(working_dir, volume):
         return container_path
 
 
+def normalize_build(service_dict, working_dir):
+    build = {}
+
+    # supported in V1 only
+    if 'dockerfile' in service_dict:
+        build['dockerfile'] = service_dict.pop('dockerfile')
```

Comment:
I think we had deleted this in a previous rebase, right? Now that it's in the V1 function